### PR TITLE
Add label customization on the Toggle

### PR DESCRIPTION
**Pull request by raphael-ms**
Add the possibility to customize the labels of the toggle, and if it's empty will keep the default OFF/ON label

### CODE REVIEW

File: packages/buffetjs-core/src/components/Toggle/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import PropTypes from 'prop-types';
 import { Toggle as StyledToggle, ToggleWrapper } from '@buffetjs/styles';
 import Label from '../Label';
 
-function Toggle({ disabled, id, className, name, onChange, value }) {
+function Toggle({ disabled, id, className, name, onChange, value, labelLeft, labelRight }) {
```
Comment (HichamELBSI) [inline packages/buffetjs-core/src/components/Toggle/index.js:13]: Hello @raphael-ms,

Can you rename the props to `rightLabel` and `leftLabel` please?

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/buffetjs-core/src/components/Toggle/index.js (hunk 1 above)
Comment (raphael-ms) [reply packages/buffetjs-core/src/components/Toggle/index.js]: yes, sorry for that. done.

Comment (MattieBelt) [reply packages/buffetjs-core/src/components/Toggle/index.js]: What about `falseLabel` & `trueLabel` or `checkedLabel` & `uncheckedLabel`? Aren't they better suited?

**Review by HichamELBSI — APPROVED** [review]
LGTM, thanks for your contribution @raphael-ms !

**Pull request merged**